src/main/java/com/google/sps/classes/AnswerObject.java

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.classes;
+
+import com.google.sps.classes.CommentObject;
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Class to create an answer for the forum. It will be used for Gson conversion.
+ */
+public class AnswerObject {
+
+  private String body;
+  private String authorName;
+  private Timestamp dateTime;
+  private int votes;
+  private List<CommentObject> commentList;
+
+  public void setBody(String body) {
+    this.body = body;
+  }
+
+  public void setAuthorName(String authorName) {
+    this.authorName = authorName;
+  }
+
+  public void setDateTime(Timestamp dateTime) {
+    this.dateTime = dateTime;
+  }
+
+  public void setVotes(int votes) {
+    this.votes = votes;
+  }  
```

Comment:
Maybe instead of a setVotes function, you could have an "incrementVotes" method? I don't imagine that the number of votes would ever increase by anything other than +1. And votes should always start at 0 upon initialization. This makes your code less error prone, in case someone accidentally resets/poorly reassigns the number of votes.

Reply:
but this is to create the object showing how many votes it already has, the voting system is not in place yet

Reply:
Ah gotcha okay I'll wait until the voting system is in place then.